packages/plugins/puppet-foreman_scap_client/puppet-foreman_scap_client.spec

```
@@ -42,8 +42,12 @@ cp -rp . %{buildroot}/%{puppet_foreman_scap_client_dir}/
 %{puppet_foreman_scap_client_dir}/lib
 %{puppet_foreman_scap_client_dir}/manifests
 %{puppet_foreman_scap_client_dir}/templates
+%exclude {puppet_foreman_scap_client_dir}/spec
```

BUILDSTDERR:     File must begin with "/": {puppet_foreman_scap_client_dir}/spec

```suggestion
%exclude %{puppet_foreman_scap_client_dir}/spec
```